src/lang/op-precedence.scrbl

```diff
@@ -0,0 +1,45 @@
+#lang scribble/base
+
+@(require "../../scribble-api.rkt")
+
+@docmodule["op-precedence" #:noimport #t #:friendly-title "Operator Precedence"]
+
+Pyret does not use implicit operator precedence or the order of operations
+that you learned in math class.  "Please Excuse My Dear Aunt Sally" does not
+apply here.
+
+Pyret disallows mixing operators without clearly defining the
+operator precedence using parentheses.  This includes all @secref["s:binop-expr"].
+
+Therefore, the following expressions are @bold{not} allowed:
+
+@pyret-block[#:style "bad-ex"]{
+1 + 1 - 1
+1 + 1 > 1
+1 + 1 == 2
+(3 * 4 / 2)
+(3 * 4) / 1 + 1
+3 * (4 / 2) + 1
+}
+
+And will raise an error like: 
+
+@(image "src/lang/binop-error.png")
+
+Conversely, these expressions are all valid in Pyret:
+
+@pyret-block[#:style "good-ex"]{
+1 + (1 - 1)
+(1 + 1) > 1
+(1 + 1) == 2
+3 * (4 / 2)
+(3 * (4 / 2))
+(3 * 4) / (1 + 1)
+(3 * (4 / 2)) + 1
+}
+
+Implicit operator precedence is a common source of errors among even
+experienced developers, so getting in the habit of explicitly defining
+precedence using parentheses is a good idea even when using languages
+that support implicit precedence.
```

Comment:
When I took over SchoolTool around 2004, I went to PyCon and met Jeff Elkner, who was trying to get an open source competency tracking system going with high school students doing most of the development, with one @pcardune being the lead student developer.  I can't remember if Paul was actually at PyCon... Anyhow, SchoolTool assimilated the competency tracking project and Paul contributed for a few years, doing some work in the summer after starting college iirc.  The direct descendant of Paul's work is still being used in about 30 counties in Virginia.  

Very happy to see Paul's carried his experiences forward to work with student developers today!